Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs

```diff
@@ -167,7 +167,7 @@ public virtual bool TryUseFood(IEntity? user, IEntity? target, UtensilComponent?
                 SoundSystem.Play(Filter.Pvs(trueTarget), UseSound, trueTarget, AudioParams.Default.WithVolume(-1f));
             }
 
-            trueTarget.PopupMessage(user, Loc.GetString("Nom"));
```

Comment:
Why did you change this?

Reply:
For reference, "Nom" is not a typo of "No", it's a common onomatopoeia for eating.

Reply:
same thing as the other one testing if the changes I did changed something in the game